app/models/concerns/user_concern.rb

```diff
@@ -172,6 +172,13 @@ def can_access_project?(project, permission: :can_view_projects)
       cached_viewable_project_ids(permission: permission).include?(project.id)
     end
 
+    def can_view_my_dashboard?
```

Comment:
I don't think this should go in the user concern actually, since it's not applicable to the warehouse users. I think we can just put it on the HMIS User class, and perhaps inline the  `can_access_staff_assignment_projects?` logic.